src/main/java/controllers/ControllerConstants.java

```diff
@@ -0,0 +1,23 @@
+package controllers;
+
+public final class ControllerConstants {
+
+    private ControllerConstants() {
+    }
+
+    public static final String EMPTY_SYMBOL = "";
+    public static final String EXPRESSION = "expression";
+    public static final String SPACES = "\\s+";
+    public static final String PLUS = "+";
+    public static final String MINUS = "-";
+    public static final String DIVIDE = "/";
+    public static final String MULTIPLY = "*";
+    public static final int ABS_RANGE = 10000;
+    public static final String OVER_RANGE = "VALUE is OVER RANGE!!!";
```

Comment:
It would be better to avoid exclamation points on error messages

Reply:
Also please deside case of message UPPER or lower. I see mix **VALUE** **is**...